src/routes/RequestsRoute.js

```diff
@@ -173,6 +177,12 @@ class RequestsRoute extends React.Component {
       path: 'cancellation-reason-storage/cancellation-reasons',
       records: 'cancellationReasons',
     },
+    staffSlips: {
+      type: 'okapi',
+      records: 'staffSlips',
+      path: 'staff-slips-storage/staff-slips?limit=100',
```

Comment:
Also, I was of the understanding that new slip templates weren’t going to use the older staff slips templates and instead use the template engine, did I misunderstand that?